b_server.md

```diff
@@ -15,245 +19,372 @@ If the server is responding returns:
+{
+  "email": "neworg@example.com",
+}
+```
 
+_Response:_
 ```json
 {
- "success": true,
- "sessionID": "5e1668d7fffaadabf0474088e..."
+ "success": true
 }
 ```
 
-{% include starsep.html %}
+Example link sent in request password email: `http://b-portal.jlinc.test/reset-password/db2506e27894aee3216273b49910062d`. The last parameter of this link is the `resetPasswordToken`.
 
-### Verifying a session ID
+Before a password is reset, the `resetPasswordToken` should be verified.
 
-_POST https://sandbox.b.jlinclabs.net/api/orgs/authn_
+_Endpoint:_
+```
+GET http://b-api.jlinc.test/api/reset_password/verify/:resetPasswordToken
+```
 
+Checks if the `resetPasswordToken` is valid and returns the email associated with the `resetPasswordToken`.
+
+_Response:_
 ```json
 {
- "session_id": "5e1668d7fffaadabf0474088e..."
+ "success": true,
+ "email": "neworg@example.com"
+}
+```
+
+Now the password can be reset.
+
+_Endpoint:_
+```
+POST http://b-api.jlinc.test/api/reset_password
+```
+
+_Post body:_
+``` json
+{
+  "resetPasswordToken": "db2506e27894aee3216273b49910062d",
+  "password": "passwordeagle",
+  "passwordConfirmation": "passwordeagle"
 }
 ```
 
-If the session is valid, updates the session time-to-live and returns true. Otherwise returns false with a 401 status.
+When the password has been successfully reset.
 
+_Response:_
 ```json
 {
  "success": true
 }
 ```
 
-{% include starsep.html %}
+<a name="authenticated-requests"></a>
+## Authenticated Requests
 
-### Getting org profile data
+These endpoints require requests to have set a `Session-Id` header with a valid session ID value.
 
-_GET https://sandbox.b.jlinclabs.net/api/orgs/profile/:sessionID_
+<a name="get-organization-profile"></a>
+### Getting an organization's profile
 
-If the session ID is valid the organization profile data will be returned:
+_Endpoint:_
+```
+GET http://b-api.jlinc.test/api/orgs/profile
+```
+
+_Response:_
 ```json
-  "success": true,
-  "organization_profile": {
+{
+  "organizationProfile": {
     "apikey": "bobco",
-    "logo": "data:image/gif;base64,R0lGODlhPQBEAPeoAJosM//AwO/…",
-    "icon": "data:image/gif;base64,R0lGODlhPQBEAPeoAJosM//AwO/…",
-    "name": "Bob & Co",
-    "dpo_email": "bobco@earthlingsunited.org",
-    "domain": "endpotatoefamine.com",
-    "contact_phone": "55555555555",
-    "contact_phone2": "5555555555",
-    "address": "83rd city ave",
-    "city": "Megatropolis",
-    "post_code": "84594",
-    "country": "The One True Country"
-  }
+    "logo": "data:image/svg+xml;charset=UTF-8,%3Csvg%20xmlns%3D%22http%3...",
+    "icon": "data:image/svg+xml;charset=UTF-8,%3Csvg%20xmlns%3D%22http%3...",
+    "name": "Bob Co",
+    "dpo_email": "neworg@example.com",
+    "domain": "bobco.com",
+    "contact_phone": "111-222-3333",
+    "contact_phone_2": "222-333-4444",
+    "address": "1 Bob St",
+    "city": "San Francisco",
+    "state": "California",
+    "post_code": "94606",
+    "country": "United States",
+    "communication_channels": {...},
```

Comment:
using `…` makes the JSON invalid. Please remove all of these and make sure the JSON is valid.